CME/admin/components/FrontMatter/Delete.php

```diff
@@ -61,33 +62,35 @@ protected function buildInternal()
 		);
 
 		$sql = sprintf(
-			'select CMEFrontMatter.id, sum(CMECredit.hours) as hours,
-				CMEProvider.title
+			'select CMEFrontMatter.id, sum(CMECredit.hours) as hours
 			from CMEFrontMatter
-				left outer join CMECredit
-					on CMECredit.front_matter = CMEFrontMatter.id
-				inner join CMEProvider
-					on CMEFrontMatter.provider = CMEProvider.id
+			left outer join CMECredit
+				on CMECredit.front_matter = CMEFrontMatter.id
 			where CMEFrontMatter.id in (%s)
-			group by CMEFrontMatter.id, CMEProvider.title',
+			group by CMEFrontMatter.id',
 			$item_list
 		);
 
-		$front_matters = SwatDB::query($this->app->db, $sql);
+		$rs = SwatDB::query($this->app->db, $sql);
 
-		foreach ($front_matters as $front_matter) {
-			$front_matter->status_level = AdminDependency::DELETE;
-			$front_matter->parent = null;
-			$front_matter->title = sprintf(
+		$class_name = SwatDBClassMap::get('CMEFrontMatter');
+
+		foreach ($rs as $row) {
+			$front_matter = new $class_name($row);
+			$front_matter->setDatabase($this->app->db);
+
+			$row->status_level = AdminDependency::DELETE;
+			$row->parent = null;
+			$row->title = sprintf(
 				CME::ngettext(
 					'%s (%s hour)',
 					'%s (%s hours)',
-					$front_matter->hours
+					$row->hours
```

Comment:
`$row->hours` can be a float, which is going to break this.

It's an existing bug, but should be fixed, and use the same pattern that we use elsewhere in for CME credits where we check explicitly for `1` to use `hour` and use `hours` for everything else, like in https://github.com/silverorange/Rap/blob/master/Rap/pages/RapFrontPage.php#L248-L253